Tests/TestingTests/IssueTests.swift

```diff
@@ -1010,6 +1010,7 @@ final class IssueTests: XCTestCase {
         return
       }
       XCTAssertFalse(issue.isKnown)
+      XCTAssertTrue(issue.severity == .error)
```

Comment:
```suggestion
      XCTAssertEqual(issue.severity, .error)
```
Similar comment for other new assertions added below in this file

Reply:
I have been spoiled by swift testing